build.gradle

```diff
@@ -0,0 +1,21 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
+    ext {
+        compose_version = '1.0.1'
+    }
+    repositories {
+        google()
+        mavenCentral()
+    }
+    dependencies {
+        classpath "com.android.tools.build:gradle:7.0.2"
+        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.30'
+
+        // NOTE: Do not place your application dependencies here; they belong
+        // in the individual module build.gradle files
+    }
+}
+
+task clean(type: Delete) {
+    delete rootProject.buildDir
+}
```

Comment:
Please, add a new line at the end of the line. (See https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b)